cmd/ctr/list.go

```diff
@@ -85,7 +126,54 @@ var listCommand = cli.Command{
 				return err
 			}
 		}
-
 		return w.Flush()
-	},
+	}
+	return nil
+}
+
+func containerListFn(context *cli.Context) error {
+	var (
+		quiet       = context.Bool("quiet")
+		ctx, cancel = appContext(context)
+	)
+	defer cancel()
+
+	client, err := newClient(context)
+	if err != nil {
+		return err
+	}
+	containers, err := client.Containers(ctx)
+	if err != nil {
+		return err
+	}
+	if quiet {
+		for _, c := range containers {
+			fmt.Printf("%s\n", c.ID())
+		}
+	} else {
+		cl := tabwriter.NewWriter(os.Stdout, 10, 1, 3, ' ', 0)
+		fmt.Fprintln(cl, "ID\tIMAGE\tRUNTIME\tSIZE")
+		for _, c := range containers {
+			var imageName string
+			if image, err := c.Image(ctx); err != nil {
+				if err != containerd.ErrNoImage {
+					return err
+				}
+				imageName = "-"
+			} else {
+				imageName = image.Name()
+			}
+			proto := c.Proto()
+			if _, err := fmt.Fprintf(cl, "%s\t%s\t%s\t%d\n",
+				c.ID(),
+				imageName,
+				proto.Runtime,
+				proto.Size(),
```

Comment:
My fault. Thought this gives container size.

I will file a new bug for printing correct container size. I think, [Container proto](https://github.com/containerd/containerd/blob/master/api/services/containers/v1/containers.proto#L30) definition should add ``Size()`` method.